submissions/SamVal007/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,127 @@
+                            </li>
+                            <li class="pop-item">
+                                <a href="#" class="popup-item"><img class="popup-item-icon" src="img/documents.png"
+                                        alt="documents">Docs</a>
+                            </li>
+                        </ul>
+                        <input type="checkbox" id="more-popup" class="more-popup">
+                        <label for="more-popup" class="more-open">More</label>
+                        <div class="more-hidden">
+                            <ul class="more-items">
+                                <li class="pop-item">
+                                    <a href="#" class="more-item"><img class="more-item-icon" src="img/chat.png"
+                                            alt="chat">Chat</a>
+                                </li>
+                                <li class="pop-item">
+                                    <a href="#" class="more-item"><img class="more-item-icon" src="img/classroom.png"
+                                            alt="classroom">Class</a>
+                                </li>
+                                <li class="pop-item">
+                                    <a href="#" class="more-item"><img class="more-item-icon" src="img/collection.png"
+                                            alt="collection">Collection</a>
+                                </li>
+                                <li class="pop-item">
+                                    <a href="#" class="more-item"><img class="more-item-icon" src="img/mars.png"
+                                            alt="mars">Mars</a>
+                                </li>
+                                <li class="pop-it                   em">
+                                    <a href="#" class="more-item"><img class="more-item-icon" src="img/blogs.png"
+                                            alt="blogs">Blogs</a>
+                                </li>
+                                <li class="pop-item">
+                                    <a href="#" class="more-item"><img class="more-item-icon" src="img/promo.png"
+                                            alt="promos">Promos</a>
+                                </li>
+                                <li><a href="https://kottans.org/" class="more-last-item kottans" target="_blank">Even
+                                        more from Kottans</a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </li>
+
+                </li>
+                <li>
+                    <a href="#" class="icon"><img src="img/bell.png" alt="bell"></a>
+                </li>
+                <li>
+                    <a href="#" class="avatar"><img src="img/Kottans^_^.jpg" alt="avatar"></a>
+                </li>
+
+
+            </ul>
+
+        </nav>
+
+    </header>
+
+
+</body>
+
+</html>
```

Comment:
If you see this symbol at Github at the end of the file, it means that the empty line is necessary. 
![empty line](https://user-images.githubusercontent.com/10871447/103133427-0b715880-46b3-11eb-971d-d7c534dac502.png)
Could you please add the empty line under the bracket?

Reply:
Yep! 
Thank You for review!